### ISSUE TITLE
Do you get IOU 67.9 on test set by the pretrained model which get 66.9 on validation set?

### ISSUE THREAD

**Issue by xizaoqu** [placeforyiming/ICCVW21-LiDAR-Panoptic-Segmentation-TradiCV-Survey-of-Point-Cloud-Cluster#5]

**Comment by placeforyiming** [comment]
> _No description provided._

For the mIoU, I directly used the semantic result provided by the Cylinder3D.

**Comment by xizaoqu** [comment]
I see, thanks

**Comment by xizaoqu** [comment]
> > _No description provided._
> 
> For the mIoU, I directly used the semantic result provided by the Cylinder3D.

I see, thanks

**Comment by xizaoqu** [comment]
> > _No description provided._
> 
> For the mIoU, I directly used the semantic result provided by the Cylinder3D.

One more question, have you figured out why thing's PQ drops so much in test set. I face the same problem. I got relatively good performance in validation set but it drops nearly 20 PQ in test set.

**Comment by placeforyiming** [comment]
> > > _No description provided._
> > 
> > 
> > For the mIoU, I directly used the semantic result provided by the Cylinder3D.
> 
> One more question, have you figured out why thing's PQ drops so much in test set. I face the same problem. I got relatively good performance in validation set but it drops nearly 20 PQ in test set.

20 is just too much, sounds like a bug, i.e. forgot to shut down the batch norm.  The test set looks more complicated than training. I visualize many frames in the test set, and there are some heavy traffic scenarios.

**Comment by xizaoqu** [comment]
> > > > _No description provided._
> > > 
> > > 
> > > For the mIoU, I directly used the semantic result provided by the Cylinder3D.
> > 
> > 
> > One more question, have you figured out why thing's PQ drops so much in test set. I face the same problem. I got relatively good performance in validation set but it drops nearly 20 PQ in test set.
> 
> 20 is just too much, sounds like a bug, i.e. forgot to shut down the batch norm. The test set looks more complicated than training. I visualize many frames in the test set, and there are some heavy traffic scenarios.

Thanks, I'll check it.

**Comment by xizaoqu** [comment]
> > _No description provided._
> 
> For the mIoU, I directly used the semantic result provided by the Cylinder3D.

I've tried the pretrained model provided by the Cylinder3D. It's mIOU on validation set is 66.9, but it's mIOU on test set is only 63.85, which is much lower than 67.9. Is there anything wrong?
